pkg/archive/archive_test.go

```diff
@@ -700,6 +701,56 @@ func tarUntar(t *testing.T, origin string, options *TarOptions) ([]Change, error
 	return ChangesDirs(origin, tmp)
 }
 
+func DecompressWithZstd(in []byte) []byte {
```

Comment:
These helper functions should not be exported, but see my other comment on using a fixture / fixed value, to prevent that we're testing if `github.com/klauspost/compress/zstd` is working as expected (which is out of scope for this test)

Reply:
Ok, I' ll un-export it.